docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -880,7 +880,7 @@ apiVersion: v1
 kind: policy
 spec:
   name: No 1Password emergency kit stored on desktop or in downloads (macOS)
-  query: SELECT 1 WHERE NOT EXISTS (SELECT 1 FROM file WHERE filename like '%%Emergency Kit%%.pdf' AND (path LIKE '/Users/%%/Downloads/%%' OR path LIKE '/Users/%%/Desktop/%%'));
+  query: SELECT EXISTS(SELECT 1 FROM file WHERE filename like '%Emergency Kit%' AND (path LIKE '/Users/%%/Downloads/%%' OR path LIKE '/Users/%%/Desktop/%%')) as does_1p_ek_exist;
```

Comment:
Minor nit: We should add the `.pdf` so this would be: `WHERE filename LIKE '%Emergency Kit%.pdf'`

Reply:
Done ✅ 

@sharvilshah I re-requested your review.